lms/djangoapps/courseware/access.py

```diff
@@ -177,15 +205,13 @@ def _can_view_courseware_with_prerequisites(user, course):  # pylint: disable=in
     Arguments:
         user (User): the user whose course access we are checking.
         course (AType): the course for which we are checking access.
-    where AType is CourseDescriptor, CourseOverview, or any other class that
-        represents a course and has the attributes .location and .id.
+            where AType is CourseDescriptor, CourseOverview, or any other class that
+            represents a course and has the attributes .location and .id.
     """
     return (
-        not settings.FEATURES['ENABLE_PREREQUISITE_COURSES']
```

Comment:
I removed it because it's checked in `_has_fulfilled_prerequisites`, right below.